packages/lesswrong/lib/collections/comments/views.ts

```diff
@@ -39,13 +44,11 @@ declare global {
    * In past versions, different subsets of these depending on whether you were
    * using an answers view, a subforum view, or something else.
    */
-  type CommentSortingMode =
-     "top"
-    |"groupByPost"
-    |"new"|"newest"
-    |"old"|"oldest"
-    |"magic"
-    |"recentComments"|"recentDiscussion"
+  export type CommentSortingMode = UnionOf<typeof COMMENT_SORTING_MODES>;
```

Comment:
This doesn't need an export since it's inside of the `declare global` block (which automatically makes it available to everything in the project)